docs/logging-and-managing-experiment-results/logging-experiment-data.rst

```diff
@@ -813,13 +817,16 @@ If your data is on AWS S3, use :meth:`~neptunecontrib.versioning.data.log_s3_dat
 
 Files
 ^^^^^
-Log any file you want, by using :meth:`~neptune.experiments.Experiment.log_artifact`. This include model_checkpoint, csv, binaries, or anything else.
+Log any file or directory you want, by using :meth:`~neptune.experiments.Experiment.log_artifact`. This include model_checkpoint, csv, binaries, or anything else.
```

Comment:
Log any file or directory you want by using :meth:`~neptune.experiments.Experiment.log_artifact`. This includes model_checkpoint, csv, binaries, or anything else.

Reply:
ok